### DISCUSSION TITLE
A lilttle more Constants:

### DISCUSSION THREAD

**Discussion by shinra358** [ikemen-engine/Ikemen-GO#1932] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

Yes, as far as I know, no real way to detect time and number of rounds per match.

### Describe the solution you'd like

to be able to detect time and final rounds
const(ikemen.timemax) = calls forth what time per round ikemen is set to.
const(ikemen.roundmax) = calls forth the number of rounds you have ikemen set to per match

useful for detecting final rounds and effects that happen when time is low. if you are in a diff fight mode, it will allocate respectively (like if youve set survival mode to be 100 rounds, it will return the value of 100)

### Additional context

examples: cvspro and sfa3 different music for final round and time almost up.

**Comment by K4thos** [comment, marked as answer]
this is already possible with GameOption trigger
